.github/workflows/tests.yml

```diff
@@ -17,7 +17,27 @@ jobs:
         run: |
           python -m pip install --upgrade pip pipenv
           pipenv install --dev
-      - name: Test
+      - name: Tests
         run: pipenv run pytest
       - name: Coveralls
-        run: pipenv run coveralls
\ No newline at end of file
+        run: pipenv run coveralls
+  linting:
+    name: Run linting
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+          architecture: x64
+      - name: Install
+        run: |
+          python -m pip install --upgrade pip pipenv
+          pipenv install --dev
+      - name: Flake8
```

Comment:
If the Makefile is set up correctly, you should just be able to run `make test` and `make lint` as the commands for these jobs. Better to have a single command that we run everywhere (locally, in CI, etc.) so we know it's always the same and if we make changes to any of the commands we don't have to update them in multiple places.

Reply:
Much better approach, I'm taking out `black` in the next commit but come back with the next PR